ckanapi/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: ckanapi
+  version: "3.5"
+
+source:
+  fn: ckanapi-3.5.tar.gz
+  url: https://pypi.python.org/packages/source/c/ckanapi/ckanapi-3.5.tar.gz
+  md5: 2ab2508df730b0e7d84e13be74adc314
+
+build:
+    number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docopt
```

Comment:
These are `run` dependencies and should be below.

PS: In my comment I mentioned `build` and not `run` :stuck_out_tongue_winking_eye: